### DISCUSSION TITLE
A huge thank you! And a couple of issues

### DISCUSSION THREAD

**Discussion by alocin** [vzhd1701/enex2notion#27] [General]
Hi, first of all I really want to thank you for this tool! I had to move more than 10k notes and the Notion importer only worked for some minor notebooks so I was considering to write my own script when I found your library. It really helped me and made the transition possible now, rather than some months ahead when I would have found the time to code. The _done_ option has also proved fundamental in my case.

Just for your information I want to highlight a couple of issues:
1) As someone else has already mentioned the import is really slow, when the Notion importer works it's a matter of a few minutes for hundreds of notes. With the enex2notion usually it's about 100 notes per hour. But I found that launching 4 concurrent imports doesn't break the usage limit and greatly reduces the total amount of time.
2) I've a lot of clipped web pages which appear quite good on Evernote so to keep the most accurate layout I tried the option to have them converted in PDF but unfortunately this started failing at a certain point, from the error message I suppose the PDF helper library was trying to fetch the images and the test from a server that not longer serves the page. So I resorted to get them in simplified html, a completely different experience, but I don't have the time to manually fix each non working note.
3) In a very few cases the number of imported notes was different from that present in Evernote. In one case it was just one and in another case it was 11 over about 1k notes. In both cases the notebooks were mainly for reference not my written content so I didn't take the time to further investigate. 
EDIT: regarding point 3 I was finishing importing the Sheet Music notebook while writing this comment and in this case  only 82 of 109 notes appeared imported. After a quick check I found the non imported notes where identical, same name and content. They were created with Evernote web clipper which some times has this kind of problems.

So thank you again!!!
Nik

**Comment by vzhd1701** [comment]
Thank you for suggestions and feedback!

Could you share some of those notes that failed to convert into PDF? `enex2notion` should remove all external links before converting to PDF, so it means I missed something while implementing that part.

**Comment by alocin** [comment]
Hi, unfortunately I don’t remember the title of those notes. I was in a
hurry and … didn’t … take note ;-)

Have a nice day,
Nik

Il giorno lun 11 apr 2022 alle 07:42 vzhd1701 ***@***.***> ha
scritto:

> Thank you for suggestions and feedback!
>
> Could you share some of those notes that failed to convert into PDF?
> enex2notion should remove all external links before converting to PDF, so
> it means I missed something while implementing that part.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vzhd1701/enex2notion/discussions/27#discussioncomment-2541975>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABBPJTDQVLAGJYIZLJZYKKLVEO3TLANCNFSM5S7M4JCA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
-- 
_____________________
*dott. Nicola De Franceschi*
Service Designer
_____________________
*Evoluzione Web*

*Contacts*
E-mail: ***@***.***
Mobile: (+39) 328 2165920
Phone: (+39) 0462 235695
Fax: (+39) 1782249664
Skype: nikdef
Web: www.evoluzioneweb.com
